change/office-ui-fabric-react-2019-09-11-10-42-22-v-mare-GroupedList-onRenderHeader-issue.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

This is a patch-level fix and doesn't need to be marked as `minor` since we aren't modifying API surface.